src/person.rb

```diff
@@ -1,9 +1,10 @@
 require_relative 'nameable'
 require_relative 'capitalize'
 require_relative 'trim'
+require_relative 'rental'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age, :parent_permission
```

Comment:
- [ ]  Same here, Kindly make sure the Getters and Setters for @rentals instance variable is present, let's add this:

```ruby
  attr_accessor :title, :author, :rentals
```
Here is a [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor%20#%20*:name%20and%20:age%20instance%20variables%20are%20the%20only%20ones%20that%20should%20be%20able%20to%20be%20modified%20that's%20why%20they%20are%20defined%20as%20attr_accessors%20(here%20both%20Getters%20and%20Setters%20are%20defined)) to the example in a previous lesson: